### ISSUE TITLE
Modpacks With Over 300 Mods Stop Generating Caves From Alex's Caves

### ISSUE THREAD

**Issue by peepeepoopoocaca** [AlexModGuy/AlexsCaves#291]
After rebuilding my modpack from scratch, I was able to verify that Deeper Caves and Terrablender are not actually incompatible with Alex's Caves. However,

1) Deeper Caves seems to be broken on its own with AC, but some other mod somehow fixes it.

2) Terrablender works without issues up until about 330-360 mods have been installed in a modpack. The number for this varies, but it seems to be related to the contents of the mods being installed. Some mods have a higher "weight" than others, and this causes the caves to stop generating at a lower number of mods. I don't know enough about modding/coding to know what exactly determines the weight of one of these mods, but I can confirm that it does exist.

NOTE: Datapacks do not seem to affect the mod limit described in the 2nd point. It is currently recommended that anyone encountering this issue should replace any mods that have a datapack version with that datapack.

Tl;dr Deeper Caves and Terrablender are not incompatible with AC, but issues can arise when a high number of mods are being used.

**Comment by AlexModGuy** [comment]
dupe #141 #288
